projects/msquic/project.yaml

```diff
@@ -2,5 +2,11 @@ homepage: "https://github.com/microsoft/msquic"
 language: c
 primary_contact: "msquicinfra@microsoft.com"
 auto_ccs:
+  - "quicdev@microsoft.com"

```

Comment:
note: This will get emails about bugs but members of this list wont be able to access them.